frontend/integration-tests/tests/kubevirt/utils.ts

```diff
@@ -85,6 +97,9 @@ export async function fillInput(elem: ElementFinder, value: string) {
       throw Error(`Failed to fill input with value: '${value}'.`);
     }
     await browser.wait(until.elementToBeClickable(elem));
+    // ensure the element is enabled, otherwise it fails to send key.
+    // https://bugs.chromium.org/p/chromedriver/issues/detail?id=1963
+    await elem.click();
```

Comment:
how does clicking on an element enable it?

Reply:
@suomiy Does it not work? I have tested it and it works well.
@rhrazdil This is how I solved `Element Not interactable` issue, if you have a better way then I will remove this.

Reply:
I am more interested about how it works. Why is the referenced chromium bug relevant?

Reply:
I think the problem here is not that an input element is disabled (because disabled element is not clickable, so clicking it won't enable it), but the input elements in wizard network/storage  tabs are in a strange state of `enabled and yet unfocusable` (Solely my description :) ).
The input field needs to be clicked twice, as the first click doesn't seem to do anything, that is why we get ElementNotInteractable and that is also why clicking the element first makes it work. 
So to sum it up, I don't think this is related to the chromium bug, because this will behave the same in gecko driver with firefox, and I don't have a better solution, because I don't think there is a generic way how to check that this is the malformed input without clicking it first. And since clicking all input elements should not do any harm, I think this is acceptable. Unless somebody has better solution

Reply:
so tables again :)

> And since clicking all input elements should not do any harm, I think this is acceptable.

yup

This should be fixed once we utilize pf4 tables so please add a TODO here and please remove the referenced bug.

Reply:
done.